tap_chromedata/streams.py

```diff
@@ -7,518 +8,327 @@
 
 from singer_sdk.streams import Stream
 import re
-#from tap_chromedata.client import chromedataStream
-#from tap_chromedata.client import aceslegacyvehiclebaseStream
-#from tap_chromedata.client import acesvehiclebaseStream
-#from tap_chromedata.client import acesvehicleconfigbaseStream
-#from tap_chromedata.client import acesvehiclemappingbaseStream
 import ftplib
 import io
 import json
 from zipfile import ZipFile
 
-# TODO: Delete this is if not using json files for schema definition
-#SCHEMAS_DIR = Path(__file__).parent / Path("./schemas")
-# TODO: - Override `UsersStream` and `GroupsStream` with your own stream definition.
-#       - Copy-paste as many times as needed to create multiple stream types.
-
-
-class QuickDataStream(Stream):
-    """Define custom stream."""
-    name = "QuickData"
-    primary_keys = ["autobuilder_style_id"]
-    replication_key = None
-    # Optionally, you may also use `schema_filepath` in place of `schema`:
-    # schema_filepath = SCHEMAS_DIR / "users.json"
-    schema = th.PropertiesList(
-        th.Property("model_year", th.IntegerType),
-        th.Property("division_name", th.StringType),
-        th.Property("model_name", th.StringType),
-        th.Property("hist_style_id", th.IntegerType),
-        th.Property("style_name", th.StringType),
-        th.Property("style_name_wo_trim", th.StringType),
-        th.Property("trim", th.StringType),
-        th.Property("full_style_code", th.StringType),
-        th.Property("style_sequence", th.IntegerType),
-        th.Property("msrp", th.NumberType),
-        th.Property("invoice", th.NumberType),
-        th.Property("destination", th.NumberType),
-        th.Property("model_effective_date", th.StringType),
-        th.Property("model_comment", th.StringType),
-        th.Property("manufacturer_name", th.StringType),
-        th.Property("manufacturer_id", th.IntegerType),
-        th.Property("division_id", th.IntegerType),
-        th.Property("hist_model_id", th.IntegerType),
-        th.Property("market_class", th.StringType),
-        th.Property("market_class_id", th.IntegerType),
-        th.Property("subdivision_name", th.StringType),
-        th.Property("subdivision_id", th.IntegerType),
-        th.Property("style_id", th.IntegerType),
-        th.Property("autobuilder_style_id", th.StringType)
-    ).to_dict()
+"""Convert camelCase words to snake_case"""
+def camel_to_snake(name):
```

Comment:
If you like, you can use the `inflection` package for this, but if it works might as well keep it